packages/features/tasker/tasks/scanWorkflowBody.ts

```diff
@@ -46,6 +61,30 @@ export async function scanWorkflowBody(payload: string) {
       continue;
     }
 
+    const timeFormat = getTimeFormatStringFromUserTimeFormat(workflowStep.workflow.user?.timeFormat);
+
+    // Determine if body is a template
+    const defaultTemplate = getTemplateBodyForAction({
+      action: workflowStep.action,
+      locale: workflowStep.workflow.user?.locale ?? "en",
+      template: workflowStep.template,
+      timeFormat,
+    });
+
+    if (
+      compareReminderBodyToTemplate({ reminderBody: workflowStep.reminderBody, template: defaultTemplate })
```

Comment:
Only call Akismet for spam detection if the reminder body is not a default template